setup.cfg

```diff
@@ -41,3 +41,40 @@ classifiers =
     Programming Language :: Python :: 3.7
     Topic :: Scientific/Engineering :: Atmospheric Science
 python_requires = >=3.7
+
+[options.entry_points]
+# top level shell commands
+console_scripts =
+    rose = metomi.rose.rose:rose
+    rosie = metomi.rose.rose:rosie
+    rosa = metomi.rose.rose:rosa [rosa]
+rose.commands =
+    app-run = metomi.rose.app_run:main
+    app-upgrade = metomi.rose.upgrade:main
+    check-software = metomi.rose.check_software:main
+    config = metomi.rose.config_cli:main
+    config-diff = metomi.rose.config_diff:main
+    config-dump = metomi.rose.config_dump:main
+    date = metomi.rose.date_cli:main
+    env-cat = metomi.rose.env_cat:main
+    host-select = metomi.rose.host_select:main
+    host-select-client = metomi.rose.host_select_client:main
+    macro = metomi.rose.macro:main
+    metadata-check = metomi.rose.metadata_check:main
+    metadata-gen = metomi.rose.metadata_gen:main
+    metadata-graph = metomi.rose.metadata_graph:main
+    namelist-dump = metomi.rose.namelist_dump:main
+    resource = metomi.rose.resource:main
+    task-env = metomi.rose.task_env:main
+    task-run = metomi.rose.task_run:main
+rosie.commands =
+    checkout = metomi.rosie.vc:checkout
```

Comment:
What's happened to `rosie-co` - I know it's only an alias, but...

Reply:
Aliases are still there, they are hardcoded in `metomi.rose.rose`.